knative-operator/pkg/common/service_monitor.go

```diff
@@ -2,27 +2,51 @@ package common
 
 import (
 	"context"
+	"fmt"
+	"os"
 
-	"github.com/openshift-knative/serverless-operator/knative-operator/pkg/apis"
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	mfclient "github.com/manifestival/controller-runtime-client"
+	mf "github.com/manifestival/manifestival"
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
 	kubemetrics "github.com/operator-framework/operator-sdk/pkg/kube-metrics"
 	"github.com/operator-framework/operator-sdk/pkg/metrics"
+	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
+	eventingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func SetupServiceMonitor(ctx context.Context, cfg *rest.Config, metricsPort int32, operatorMetricsPort int32) error {
+const (
+	EventingBrokerServiceMonitorPath     = "deploy/resources/broker-service-monitors.yaml"
+	EventingSourceMonitorPath            = "deploy/resources/source-service-monitor.yaml"
```

Comment:
EventingBrokerService   MonitorPath 

vs

EventingSource  MonitorPath 

let's do EventingSourceService path ?  see: https://github.com/openshift-knative/serverless-operator/pull/467/files#diff-96b561fd4c465bd07c16d9f0f812bdb1R32

Reply:
These are the paths to the monitors only. I had to split them because I need to create first the service (it has a different yaml and path) and then get the object to pass it to the service monitor manifest to have a proper UID for the owner references stuff.  Could you elaborate more?

Reply:
I meant more on the varible naming    EventSourceSerivice  (to be consistent)

Reply:
should we change `EventingSourceMonitorPath` name to `EventingSourceServiceMonitorPath` ?  the `EventingBroker....` the line above does have the `Service` in the name (for consistency)

Reply:
yeah sure